liquibase.gradle

```diff
@@ -0,0 +1,31 @@
+configurations {
+  liquibase
+}
+
+dependencies {
+  liquibase group: 'org.liquibase.ext', name: 'liquibase-hibernate5', version: 3.6
+}
+
+//loading properties file.
+Properties liquibaseProps = new Properties()
+liquibaseProps.load(new FileInputStream("$projectDir/src/main/resources/liquibase.properties"))
+
+task migratePostgresDatabase(type: JavaExec) {
+  group = "liquibase"
+
+  classpath sourceSets.main.runtimeClasspath
+  classpath configurations.liquibase
+  main = "liquibase.integration.commandline.Main"
+
+  def urlString = project.hasProperty("dburl") ? "jdbc:postgresql://$dburl" : liquibaseProps.getProperty('url')
+  def user = project.hasProperty("flyway.user") ? "${rootProject.properties['flyway.user']}" : liquibaseProps.getProperty('username')
+  def password = project.hasProperty("flyway.password") ? "${rootProject.properties['flyway.password']}" : liquibaseProps.getProperty('password')
+
+  args "--changeLogFile=./src/main/resources/db/db-changelog.yaml"
```

Comment:
If we would like to have some test data on environments after deployment (e.g. preview) how we would like to approach it? Another gradle task for test data or maybe some logic on this task to check if it's not production evnironment?